### DISCUSSION TITLE
Add AI Chatbot button into Zen Sidebar from Firefox Labs

### DISCUSSION THREAD

**Discussion by m3957** [zen-browser/desktop#1243] [Ideas]
When enabling the AI chatbot, the only way you can access it is by clicking on the bookmarks button and switching to the AI tab. It would be useful to have a little star icon that opens the AI stuff.

**Comment by KiKaraage** [comment]
You can put a Sidebar button to the navbar from the Customize Toolbar button btw...
![image](https://github.com/user-attachments/assets/e4c60e56-80ea-40f8-bdca-5862a04a3fa0)

**Reply by astroboii47** [reply]
The AI chatbot seems to open separate to the Sidebar so that button just toggles the web panel sidebar option at the moment
![CleanShot 2024-09-05 at 21 10 10@2x](https://github.com/user-attachments/assets/33168f2a-72e4-481e-b5e4-22687d60212b)

**Reply by astroboii47** [reply]
Actually nevermind you're right - the icons just look identical but they open different panels lol 
![CleanShot 2024-09-05 at 21 12 51@2x](https://github.com/user-attachments/assets/c9e81737-ad04-4f01-80ce-9c435940ddf4)

**Reply by m3957** [reply]
Yeah it works but I think a little ✨ button in the sidebar would be better.

**Reply by astroboii47** [reply]
Yep I agree for sure! Was just noticing that the button does work but it's not intuitive at all cos it's the same icon haha

**Comment by tommdq** [comment]
Does the AI Chatbot sidebar function have a shortcut ? I don't see it on the Sidebar Actions Keyboard Shortcut menu.

**Comment by NyllRE** [comment]
This should be a normal thing to have. just the option to add a keyboard shortcut or a button to use

**Comment by trey-a-12** [comment]
To those wondering how to enable the AI Sidebar functionality, go to about:config and edit the settings to these values ([per this Reddit post's advice](https://www.reddit.com/r/zen_browser/comments/1jb1kxs/how_to_restore_the_ai_chatbot_sidebar/)). Once enabled, press Ctrl + X to toggle the sidebar, or right-click selected text and use the "Ask [Chatbot]" context menu item.

<img width="1080" height="712" alt="87392" src="https://github.com/user-attachments/assets/04b70156-41ca-4d1c-9e73-ef6624d1b41d" />

In Zen 1.17.3b (Firefox 144.0) (aarch64), there is currently no toolbar button to enable this sidebar to my knowledge. Either invoking the keyboard shortcut or right-clicking the selected text with the "Ask [Chatbot]" button is necessary.
